airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLWriterTest.kt

```diff
@@ -114,9 +135,21 @@ class MSSQLDataCleaner : DestinationCleaner {
     }
 }
 
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
actually derp, I missed this on my readthrough - why do we need this? I'm pretty sure this is the problem behind https://github.com/airbytehq/airbyte/pull/51594#issuecomment-2596928215

(i.e. if you remove this, it should work as expected, b/c the default is PER_METHOD)